core/src/main/resources/hudson/cli/Messages_it.properties

```diff
@@ -4,4 +4,4 @@ DeleteJobCommand.ShortDescription=Cancella un job
 OnlineNodeCommand.ShortDescription=Resume using a node for performing builds, to cancel out the earlier "offline-node" command.
 
 ClearQueueCommand.ShortDescription=Pulisce la coda di lavoro
-
+DisconnectNodeCommand.ShortDescription=Disconnects from a node
```

Comment:
@pjanouse I think it was wrong that they were in there to begin with. Probably a half-finished translation. Only strings that don't appear in the localized resource will be considered untranslated by tooling, so copying the English version into other languages confuses things.